qiskit/assembler/assemble_schedules.py

```diff
@@ -203,6 +203,11 @@ def _assemble_instructions(
                 name=name)
             user_pulselib[name] = instruction.pulse.samples
 
+        # ignore delay instrs on acq channels (don't matter w/ absolute timing)
+        if (isinstance(instruction, instructions.Delay) and
+                isinstance(instruction.channel, channels.AcquireChannel)):
```

Comment:
If the comment was true, and delays didn't matter with absolute timing, then this half of the condition could be removed, and we could ignore all Delays

Reply:
Addressed in https://github.com/Qiskit/qiskit-terra/pull/6000/commits/e24adf163f5804576f2d85c7c5bf8a895238775b.